packages/recomponents/src/components/r-button/r-button.vue

```diff
@@ -8,7 +8,9 @@
        :target="$attrs.href ? '_target' : $attrs.target || ''"
     >
         <!-- @slot Text content inside button  -->
-        <slot>Link</slot>
+        <div class="capitalize-the-first-letter">
```

Comment:
rather than wrapping the slot in a div, can you add the class directly to the `a` or `button` element? You can add it directly to the `classes` computed property

Reply:
I think it's better to use static class attribute instead of classes since we apply first letter capitalisation by default for all buttons

Reply:
> can you add the class directly to the a or button element?

In r-icon-button, the first letter could be an icon(svg HTML tag), that's why I wrapped the slot in a div. but for r-button, your suggestion is right. I just keep the code consist.

See more discussion here: https://github.com/Rebilly/rebilly-recomponents/pull/183#discussion_r425711551

Reply:
> better to use static class attribute instead of classes

I don't quite understand it here, but I improved it, added one more prop: `capitalizeFirstLetter`

![2020-05-18 12-34-44 2020-05-18 12_34_59](https://user-images.githubusercontent.com/4344909/82174700-13d70900-9904-11ea-85d3-d5b4d88314e8.gif)